grakn-engine/src/test/java/ai/grakn/engine/EngineTestHelper.java

```diff
@@ -0,0 +1,68 @@
+package ai.grakn.engine;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+/**
+ * <p>
+ * This class instantiates an engine server-side component once, statically until the JVM exists.
+ * Any state cleanup needs to be done by tests themselves - more code, more work, but more performant
+ * test suite. So all unit tests will share the same components and will be written with that in mind.  
+ * </p>
+ * 
+ * @author borislav
+ *
+ */
+public class EngineTestHelper {
```

Comment:
Anyway, @fppt let me know if you'd rather put this class in the test tools module. Everything I did with this PR initially was with the goal to communicate the approach (better with code than lots of explanations). We can move things around now.